src/main/java/it/gov/pagopa/pu/sil/connector/paymentnotification/config/NativePaymentNotificationApisHolder.java

```diff
@@ -0,0 +1,52 @@
+package it.gov.pagopa.pu.sil.connector.paymentnotification.config;
+
+import it.gov.pagopa.paymentnotification.controller.ApiClient;
+import it.gov.pagopa.paymentnotification.controller.generated.DefaultApi;
+import it.gov.pagopa.pu.sil.config.agid.AgidDataIntegrityInterceptor;
+import it.gov.pagopa.pu.sil.config.agid.PuIntegrityDataConfig;
+import it.gov.pagopa.pu.sil.config.rest.RestTemplateConfig;
+import jakarta.annotation.PreDestroy;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Component
+public class NativePaymentNotificationApisHolder {
```

Comment:
let's not use `Native` prefix! it's implicit
```suggestion
public class PaymentNotificationApisHolder {
```